packages/wallet-sdk/src/sign/scw/SCWSigner.ts

```diff
@@ -78,30 +85,62 @@ export class SCWSigner implements Signer {
     this.handleResponse(args, decrypted);
   }
 
-  async request(request: RequestArguments) {
+  // TODO: Properly type the return value
+  async request(request: RequestArguments): Promise<any> {
     if (this.accounts.length === 0) {
       switch (request.method) {
         case 'wallet_switchEthereumChain': {
           assertParamsChainId(request.params);
           this.chain.id = Number(request.params[0].chainId);
           return;
         }
-        case 'wallet_connect':
-        case 'wallet_sendCalls':
+        case 'wallet_connect': {
```

Comment:
This and the `wallet_connect` below look the same. Maybe we can create a util to share between the two